src/studio/nuxeo-online-services-portal/token-management.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Token Management
+description: Personal authentication tokens can be generated from the Nuxeo Online Services portal.
+review:
+    comment: ''
+    date: ''
+    status: ok
+toc: true
+---
+
+## Token Generation
+Personal authentication tokens can be generated from the Nuxeo Online Services portal.
+
+Tokens can be used as a replacement for a password, and can be revoked anytime. Here are a few situations where they can be useful:
```

Comment:
I would rather say that they _are_ used as a replacement for passwords, which incidentally are to be used _only_ when logging in nos. => This means passwords usage in APIs/Maven access is deprecated and will be unsupported at some point :)

Reply:
OK, changed it to not even talk about the password => simpler and people will take the habit sooner hopefully.